wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Interactive.kt

```diff
@@ -35,13 +37,17 @@ internal constructor(
    */
   suspend fun withdraw(
     accountAddress: String,
-    assetId: IssuedAssetId,
+    assetId: StellarAssetId,
     authToken: AuthToken,
     extraFields: Map<String, String>? = null,
     fundsAccountAddress: String? = null,
   ): InteractiveFlowResponse {
     return flow(accountAddress, assetId, authToken, extraFields, fundsAccountAddress, "withdraw") {
-      it.withdraw[assetId.code]
+      if (assetId is IssuedAssetId) {
```

Comment:
nit: can replace lambda type with:
`assetGet: (AnchorServiceInfo, String) -> AnchorServiceAsset?`
second argument is either assetId.code or NativeAssetId.id

Reply:
I'll leave it as is to minimize the changes